rfcs/meeting-facillitation.md

```diff
@@ -0,0 +1,27 @@
+# Meeting Facillitation
+
+This document aims to formalize our meeting structure with respect to running them.
+
+-----
+
+Each meeting will have a facillitator and a note taker who sign up for the role for each meeting
+ahead of time. These roles are open to all Community Team members. The responsibilities for each are
+listed below.
+
+### Facillitator
+
+- On the Monday before a team meeting an agenda issue will be issued.
+  - This is done by copying the previous weeks agenda and removing unnecessary bits and then opening
```

Comment:
we could also have a template which seems like the less complicated solution to me

Reply:
See https://github.com/rust-community/content-team/blob/master/ISSUE_TEMPLATE/meeting_agenda.md

Reply:
Here's our own version https://github.com/rust-community/team/blob/master/ISSUE_TEMPLATE/meeting_agenda.md